src/pages/Graph/SummaryLink.tsx

```diff
@@ -127,10 +127,10 @@ const getLink = (nodeData: GraphNodeData, nodeType?: NodeType) => {
 
 export const renderBadgedHost = (host: string) => {
   return (
-    <span>
+    <div>
       <PFBadge badge={PFBadges.Host} />
-      {host}
-    </span>
+      {host === "*" ? "(All hosts)" : host}
```

Comment:
Same here, I like keeping the `*` as that reflects the actual config entry (right?).

Reply:
Right, the star reflects the config entry.

I'm taking your suggestion of keeping both: `* (all hosts)`